### PR TITLE
Fine tune more downdrop settings, use SVG for labels, improve Repo Topic Edit form

### PR DESCRIPTION
**Pull request by wxiaoguang**
Although it seems that some different purposes are mixed in this PR, however, they are all related, and can be tested together, so I put them together to save everyone's time.

Diff: `+79 −84`, everything becomes much better.

### Improve the dropdown settings.

Move all fomantic-init related code into our `fomantic.js`

Fine-tune some dropdown global settings, see the comments.

Also help to fix the first problem in #23625 , cc: @yp05327 

The "language" menu has been simplified, and it works with small-height window better.

### Use SVG instead of `<i class="delete icon">`

It's also done by `$.fn.dropdown.settings.templates.label` , cc: @silverwind 

### Remove incorrect `tabable` CSS class

It doesn't have CSS styles, and it was only in Vue. So it's totally unnecessary, remove it by the way.

### Improve the Repo Topic Edit form

* Simplify the code
* Add a "Cancel" button
* Align elements

Before:

<details>

image

</details>

After:

image

### CODE REVIEW

**Comment by silverwind** [comment]
> It's also done by $.fn.dropdown.settings.templates.label , cc: @silverwind

Nice, one step closer to removing FontAwesome.

**Comment by wxiaoguang** [comment]
> Do whe know which other parts of Fomantic are using icon fonts?

I have no idea at the moment. Maybe in the future, grepping the Fomantic source code would give some answers.

**Comment by silverwind** [comment]
I imagine we need to remove the `icon` module when we do that, and I guess that module is imported in various other fomantic modules.

File: web_src/js/features/repo-home.js [hunk 1]
```diff
@@ -6,27 +6,29 @@ const {appSubUrl, csrfToken} = window.config;
...
-    const hidePrompt = $('#validate_prompt');
-    const prompts = {
-      countPrompt: hidePrompt.children('#count_prompt').text(),
-      formatPrompt: hidePrompt.children('#format_prompt').text()
-    };
-    hidePrompt.remove();
-    return prompts;
-  }
+  $('#cancel_topic_edit').on('click', () => {
+    hideElem(editDiv);
```
Comment (lafriks) [inline web_src/js/features/repo-home.js:28]: Shouldn't it cancel also changes done?

Comment (wxiaoguang) [reply web_src/js/features/repo-home.js:28]: It needs a lot (I guess) of new code, so I didn't do that. And at the moment, in the Issue Title Edit form, the cancel button doesn't reset the changes either.

If it must be improved, I guess it's better to improve them together (later, in a separate PR)

At least, with this PR, their behaviors are consistent.

**Review by lunny — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Pull request merged**